src/api/annotations.ts

```diff
@@ -0,0 +1,36 @@
+import {AnnotationPriorities} from 'components/consts';
+
+export interface RangeAPIModel {
+  start: string;
+  startOffset: number;
+  end: string;
+  endOffset: number;
+}
+
+export interface AnnotationAPIPostModel {
```

Comment:
I would say we at least one abstraction layer away from that HTTP, so what about referring to model/resource life cycle and using _create_ or something like that?

Reply:
Actually I didn't mean HTTP POST method (but the *post* action) but I agree that `create` is better, I'll change it

Reply:
OK

Off-topic: what's the difference between post action and method for you here? What do you mean by _action_ ?